docs/topics/request-response.rst

```diff
@@ -24,7 +24,7 @@ below in :ref:`topics-request-response-ref-request-subclasses` and
 Request objects
 ===============
 
-.. class:: Request(url[, callback, method='GET', headers, body, cookies, meta, encoding='utf-8', priority=0, dont_filter=False, errback])
+.. class:: Request(url[, callback, method='GET', headers, body, cookies, meta, encoding='utf-8', priority=0, dont_filter=False, errback, flags = null])
```

Comment:
I'm confused. [`Request` doesn't have a `flags` parameter](https://github.com/scrapy/scrapy/blob/73a5571c6044d5aea47b4f973e325c2f3d4e25dc/scrapy/http/request/__init__.py#L19). Moreover, I think you meant `flags=None`, as Python doesn't have `null`.